source/windows/secure_channel_tls_handler.c

```diff
@@ -1111,6 +1111,12 @@ static int s_do_application_data_decrypt(struct aws_channel_handler *handler) {
                         "id=%p: Decrypt ended exactly on the end of the record, resetting buffer.",
                         (void *)handler);
                 }
+            } else {
+                AWS_LOGF_ERROR(
+                    AWS_LS_IO_TLS,
+                    "id=%p: Error decrypting message. Unexpected type of output buffer.",

```

Comment:
ask henso about this case
if this is the right way to handle it, or if he intended something else to happen